contracts/collectxyz-nft-contract/src/execute.rs

```diff
@@ -293,12 +293,13 @@ pub fn migrate(deps: DepsMut, _env: Env, _msg: MigrateMsg) -> StdResult<Response
     let num_tokens = 1 + cw721_contract.token_count(deps.storage)?;
     for token_num in 1..num_tokens {
         let token_id = format!("xyz #{}", token_num);
-        tokens().update(deps.storage, &token_id, |old_token| match old_token {
-            Some(old_token) => {
-                let mut new_token = old_token.clone();
-                new_token.image = Some(base64_token_image(&old_token.extension.coordinates));
-                Ok(new_token)
-            }
+        let new_token_id = format!("#{}", token_num);
```

Comment:
This format string would give new token ids the format `"#123"`, when we want `123`:
```suggestion
        let new_token_id = format!("{}", token_num);
```